### PR TITLE
docs: update the org readme

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -106,9 +113,24 @@ OpenEBS is the most successful Stateful Persistent Container Native Storage plat
...
+Meeting Link: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1
+<br>
+Starting in August 2024, the meetings will be recorded and posted on YouTube.
+
+## Other Resources
+Maintainers meet weekly and minute their meetings. A public record of the meetings is here.
+See [Vision](https://github.com/openebs/community/blob/develop/VISION.md) for more detail on what we do, [Governance](https://github.com/openebs/community/blob/develop/GOVERNANCE.md) for how we do it, and [Contributing](https://github.com/openebs/community/blob/develop/CONTRIBUTING.md) for how to get involved
+
+## New Contributors and Maintainers
+We are actively looking for new contributors and maintainers. Want to get involved, but not sure how? Please email the OpenEBS maintainers.
```
Comment (Abhinandan-Purkait) [inline profile/README.md]: @edrob999 We need to specify the email as well right?

Comment (tiagolobocastro) [reply profile/README.md]: Yes we do

**Review by tiagolobocastro — APPROVED** [review]

**Review by niladrih — COMMENTED** [review]
Could you please update the author
```
git commit --amend --author "Ed Robinson <ed.robinson@gmail.com>" -s -m "docs: update the org readme"
```

File: profile/README.md (hunk 1 above)
Comment (edrob999) [reply profile/README.md]: Yes. Feedback from TAG Contributor Strategy Governance, is to make it very-very-clear how you would initiate becoming a maintainer "email this address"

**Review by edrob999 — APPROVED** [review]
Thanks for making these changes so quickly

**Review by niladrih — APPROVED** [review]

**Review by avishnu — APPROVED** [review]

**Pull request merged**